src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -1628,8 +1628,8 @@ CmdTechDrawExtensionShortenLine::CmdTechDrawExtensionShortenLine()
 {
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Shorten Line");
-    sToolTipText = QT_TR_NOOP("Shorten a cosmetic line or centerline at both ends:<br>\
+    sMenuText = QT_TR_NOOP("Shorten line");
+    sToolTipText = QT_TR_NOOP("Shortens a selected cosmetic line or centerline at both ends:<br>\
```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Shortens a selected cosmetic line or centerline at both ends by the specified delta distance")
```
then remove next 3 lines